### PR TITLE
Announcement post for Istio 1.6

### PR DESCRIPTION
**Pull request by oaktowner**
Release announcement for Istio 1.6.

### CODE REVIEW

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ability to control your trace sampling rates, and updated Grafana dashboards
+(and we're even publishihg them on 'grafana.com' on the [Istio org page]
+(https://grafana.com/orgs/istio)).
+
+## Other improvements
+There are great traffic management features (like supporting the experimental
+Kubernetes Service APIs, better support for Ingress, and better header
+handling).
+
+For those of you who are adding non-Kubernetes workloads to meshes (for
```
Comment (rshriram) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:67]: we might want to make this a bit more prominent

Comment (rshriram) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:67]: We created an API called WorkloadEntry to elevate a VM or bare metal machine to the same level as a kubernetes pod, to give it first class representation in Istio, instead of just a dumb endpoint with some IP address. We now have the ability to define a Service that is backed by both Pods and VMs. Users now have the ability to have a heterogeneous mix of deployments (VMs and Pods) for the same service, providing a great way for migrating VM workloads to the mesh. 

or something along those lines.. and this can be called out on its own..

Comment (rshriram) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:67]: cc @smawson

Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:68]: Great suggestion -- that was the detail I needed to give it its own section. Doing that now!

**Comment by oaktowner** [comment]
I'm a little confused by the lint error I'm getting. It says 

> "Building sites … WARN 2020/05/15 16:47:10 Content directory "/work/content/en/docs/setup/upgrade" have both index.* and _index.* files, pick one." 

but I haven't touched that file.

File: .spelling [hunk 2]
```diff
@@ -701,6 +701,7 @@ wildcard
 wildcarded
 wildcards
 workgroup
+WorkloadEntry
```
Comment (sdake) [inline .spelling:704]: please use backticks in the documentation for `WorkloadEntry` and remove this entry from the `.spelling` file as per various documentation guides.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+Here’s some of what’s coming to you in today's release:
+
+## Simplify, simplify, simplify
+
+Last release, we introduced **Istiod**, a component that reduced the number of
+components in an Istio installation by combining the functionality of several
+other components. In Istio 1.6, you'll find that Citadel, the sidecar
+injector and Galley are no longer deployed.
+
+You'll also find that we are using the new `AppProtocol` API from Kubernetes.
```
Comment (sdake) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:31]: `appProtocol` is not an API, rather an internal field in the the EndpointPort API.

A better way to write this sentence would be
```
"You'll also find that we are using the new `appProtocol` field in the [EndpointPort](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#endpoints-v1-core) API.```

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+What does that mean for you? It means that you won't need to use the name
+field in your service to denote the protocol -- a simpler user experience.
+
+## Better lifecycle
+
+We continue to make installing and upgrading Istio a better experience. Our
+command line tool `istioctl` gives better diagnostic information, has a simpler
+install command, and even gives status in color!
```
Comment (sdake) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:39]: Lovin it 👍

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+## Observe this
+
+Many companies adopt Istio solely to get better observability of distributed
+applications, so we continue to invest there. You'll have to read the release
+notes to see all of the features, but you'll see more configurability, better
+ability to control your trace sampling rates, and updated Grafana dashboards
+(and we're even publishing them on [Grafana](https://grafana.com) on the
+[Istio org page](https://grafana.com/orgs/istio)).
+
+## Better VM support
```
Comment (sdake) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:62]: might break down the acronym. I think its safe to say everyone knows what VM means, however, blog posts should be free of acronyms if possible.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+workloads first-class representation in Istio. It elevates a VM or bare metal
+workloads to the same level as a Kubernetes pod, instead of just an endpoint
+with an IP address. You now even have the ability to define a Service that is
+backed by both Pods and VMs. Why is that useful? Well, now you now have the
+ability to have a heterogeneous mix of deployments (VMs and Pods) for the same
+service, providing a great way for migrating VM workloads to the mesh.
+
+Expanding support for VM-based workloads was another theme we called out
+in our roadmap post, and you can expect to see more in this area over the
+coming releases.
```
Comment (sdake) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:76]: 💯 🥇 👍

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+field in your service to denote the protocol -- a simpler user experience.
+
+## Better lifecycle
+
+We continue to make installing and upgrading Istio a better experience. Our
+command line tool `istioctl` gives better diagnostic information, has a simpler
+install command, and even gives status in color!
+
+Upgrading Istio has been improved as well, in several powerful ways. First, we
+now support canarying of the Istio control plane itself. That means you can
```
Comment (frankbu) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:42]: How about putting a link to @howardjohn's blog post here?

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 1 above)
Comment (frankbu) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:63]: There's also a blog post PR about this that could be linked to from the announce post. See #7321

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 5 above)
Comment (sdake) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:62]: after line 62, an empty line is needed for proper markdown formatting - to ensure the linter passes and the document renders properly.

File: .spelling (hunk 2 above)
Comment (oaktowner) [reply .spelling:704]: done

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 3 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:31]: great suggestion. done.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 5 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:62]: Done.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+our [roadmap post](/blog/2020/tradewinds-2020/), sailing toward more
+simplicity, a better installation experience, and some other goodies as well.
+
+Here’s some of what’s coming to you in today's release:
+
+## Simplify, simplify, simplify
+
+Last release, we introduced **Istiod**, a component that reduced the number of
+components in an Istio installation by combining the functionality of several
+other components. In Istio 1.6, you'll find that Citadel, the sidecar
```
Comment (howardjohn) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:28]: nit: To the unfamiliar this may sound like we are removing more things in 1.6 as part of the default install. For most users these were already removed in 1.5. Now there is just no way to configure them at all.

Maybe we can rephrase this to something along the lines of "In Istio 1.6 we have completed this transition and have fully moved functionality into Istiod, allowing us to remove x y z" or similar?

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 3 above)
Comment (howardjohn) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:31]: nit: Kubernetes users don't know what EndpointPort is, its an internal name. We can just clarify that its for selecting the protocol in Service. And maybe link to https://preliminary.istio.io/docs/ops/configuration/traffic-management/protocol-selection/#manual-protocol-selection instead of k8s api where we explain how and why to use this

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 7 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:43]: It's not published yet...happy to enlinkify it when that's published.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 8 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:28]: Great suggestion. Done.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 5 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md]: done!

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+themselves).
+
+Check out the [documentation](/docs/setup/upgrade/) for all of the details on
+the new upgrade experience.
+
+## Observe this
+
+Many companies adopt Istio solely to get better observability of distributed
+applications, so we continue to invest there. You'll have to read the release
+notes to see all of the features, but you'll see more configurability, better
```
Comment (smawson) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:59]: Consider rephrasing to something like "there are too many changes to list them all here, so see the release notes for the full details"

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+[Istio org page](https://grafana.com/orgs/istio)).
+
+## Better Virtual Machine support
+
+For those of you who are adding non-Kubernetes workloads to meshes (for
+example, workloads deployed on VMs), the new
+[`WorkloadEntry`](/docs/reference/config/networking/workload-entry/) resource
+makes that easier than ever. We created this API to give non-Kubernetes
+workloads first-class representation in Istio. It elevates a VM or bare metal
+workloads to the same level as a Kubernetes pod, instead of just an endpoint
```
Comment (smawson) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:71]:
```suggestion
workload to the same level as a Kubernetes pod, instead of just an endpoint
```

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+with an IP address. You now even have the ability to define a Service that is
+backed by both Pods and VMs. Why is that useful? Well, now you now have the
```
Comment (smawson) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:73]:
```suggestion
backed by both Pods and VMs. Why is that useful? Well, now you have the
```

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+ability to have a heterogeneous mix of deployments (VMs and Pods) for the same
+service, providing a great way for migrating VM workloads to the mesh.
```
Comment (smawson) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:75]:
```suggestion
service, providing a great way to migrate VM workloads to the mesh.
```

Comment (smawson) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:75]: This is actually a little weird, since sharing a service across pods and vms isn't really about migrating to the mesh (they're already both in the mesh). Maybe we mean migrating to containers? Or we can just leave it more open and not describe an example use case?

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+
+Expanding support for VM-based workloads was another theme we called out
```
Comment (smawson) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:77]: Maybe we can pull the first part of this up to the beginning of the section? Otherwise it reads a bit weird. Or reword somehow...

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
+in our roadmap post, and you can expect to see more in this area over the
+coming releases.
+
+## Other improvements
+
+There are great traffic management features (like supporting the experimental
```
Comment (smawson) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:83]: This feels like an afterthought, needs slightly more love.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 15]
```diff
@@ -0,0 +1,99 @@
+---
+title: Announcing Istio 1.6
+linktitle: 1.6
+subtitle: Major Update
+description: Istio 1.6 release announcement.
+publishdate: 2020-05-19
```
Comment (brian-avery) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:6]: This should be the 21st

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 12 above)
Comment (rshriram) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:75]: I like the migrating to kubernetes part

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+simplicity, a better installation experience, and some other goodies as well.
+
+Here’s some of what’s coming to you in today's release:
+
+## Simplify, simplify, simplify
+
+Last release, we introduced **Istiod**, a component that reduced the number of
+components in an Istio installation by combining the functionality of several
+other components. In Istio 1.6, we have completed this transition and have fully
+moved functionality into Istiod, allowing us to remove Citadel, the sidecar
```
Comment (brian-avery) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:29]: this sentence sounds weird.

```suggestion
moved functionality into Istiod/ This has allowed us to remove Citadel, the sidecar injector, and Galley.
```

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 17]
```diff
@@ -0,0 +1,99 @@
...
+upgrade in your clusters (still giving you the control over updating the proxies
+themselves).
+
+Check out the [documentation](/docs/setup/upgrade/) for all of the details on
+the new upgrade experience.
+
+## Observe this
+
+Many companies adopt Istio solely to get better observability of distributed
+applications, so we continue to invest there. You'll have to read the release
```
Comment (brian-avery) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:58]: should this link to the release notes?

Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:58]: YES! Will fix.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 9 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:59]: done

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 18]
```diff
@@ -0,0 +1,99 @@
...
+## Join the Istio community
+
+As always, there is a lot happening in the
+[Community Meeting](https://github.com/istio/community#community-meeting);
+join us every other Thursday at 11 AM Pacific. We'd love to have you join the
+conversation at [Istio Discuss](https://discuss.istio.io), and you can also join
+our [Slack channel](https://istio.slack.com).
+
+We were very proud to be called out as one of the top five
+[fastest growing](https://octoverse.github.com/#top-and-trending-projects)
```
Comment (brian-avery) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:96]: This is cool! Hadn't realized that.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 10 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:71]: I rephrased. Have had a customer specifically talk about the desire to use this to help migrate workloads from vms to clusters without disrupting traffic to/from. Check it out.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 12 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:75]: done.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 13 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:77]: I split it into two. PTAL.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 15 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:6]: fixed

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 17 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:58]: Fixed, and also there will be a link at the top autogenerated like the table you see on https://istio.io/news/releases/1.5.x/announcing-1.5/

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 14 above)
Comment (craigbox) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:89]: Do we have the first implementation of the Service APIs? That feels worth calling out

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 19]
```diff
@@ -0,0 +1,105 @@
...
+
+## Simplify, simplify, simplify
+
+Last release, we introduced **Istiod**, a component that reduced the number of
+components in an Istio installation by combining the functionality of several
+other components. In Istio 1.6, we have completed this transition and have fully
+moved functionality into Istiod. This has allowed us to remove Citadel, the
+sidecar injector, and Galley.
+
+You'll also find that we are using the new `appProtocol` field in the Kubernetes
```
Comment (craigbox) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:32]: Should we lead with the benefit, rather than the implementation detail?

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 20]
```diff
@@ -0,0 +1,105 @@
...
+Expanding our support for workloads not running in Kubernetes was one of the
+our major areas of investment for 2020, and we're excited to announce some
+great progress here.
+
+For those of you who are adding non-Kubernetes workloads to meshes (for
+example, workloads deployed on VMs), the new
+[`WorkloadEntry`](/docs/reference/config/networking/workload-entry/) resource
+makes that easier than ever. We created this API to give non-Kubernetes
+workloads first-class representation in Istio. It elevates a VM or bare metal
+workload to the same level as a Kubernetes pod, instead of just an endpoint
```
Comment (craigbox) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:77]: s/pod/Pod

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 19 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:32]: Great suggestion. Done.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 7 above)
Comment (frankbu) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:46]: It's in preliminary, targeted to go out with the release. https://preliminary.istio.io/blog/2020/multiple-control-planes/

**Comment by sdake** [comment]
@oaktowner thanks for addressing my comments. LGTM!

**Comment by sdake** [comment]
@oaktowner about the broken link in lint to upgrades - don't worry about that, that gets fixed by @frankbu in the magic that is the actual release of the project.

Since you have no other linting problems, the rest looks great.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 21]
```diff
@@ -0,0 +1,107 @@
...
+    - /news/announcing-1.6
+---
+
+We are pleased to announce the release of Istio 1.6!
+
+{{< relnote >}}
+
+With this release, we are continuing the path we charted earlier this year in
+our [roadmap post](/blog/2020/tradewinds-2020/), sailing toward more
+simplicity, a better installation experience, and some other goodies as well.
```
Comment (craigbox) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:20]: and adding some other goodies (needs to be active voice)

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 22]
```diff
@@ -0,0 +1,107 @@
...
+
+Here’s some of what’s coming to you in today's release:
+
+## Simplify, simplify, simplify
+
+Last release, we introduced **Istiod**, a component that reduced the number of
+components in an Istio installation by combining the functionality of several
+other components. In Istio 1.6, we have completed this transition and have fully
+moved functionality into Istiod. This has allowed us to remove Citadel, the
```
Comment (craigbox) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:29]: "remove the separate Deployments for"; to make it clear the functionality remains

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md [hunk 23]
```diff
@@ -0,0 +1,107 @@
...
+sidecar injector, and Galley.
+
+Great news! We've got a simplified experience for developers who are taking
```
Comment (craigbox) [inline content/en/news/releases/1.6.x/announcing-1.6/_index.md:32]: This is only Alpha in Kubernetes 1.18 and so sounds a little overselling to me.

What about "As Kubernetes advanced and adds support for defining protocols, Istio will be able to learn from this too. In the upcoming 1.19 release, support for defining an `AppProtocol` on a `Service` will mean you will no longer need to append the name field  to denote the protocol.

Better link is also https://github.com/kubernetes/enhancements/issues/1507

Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:32]: I'll take your link and rephrase to include alpha. But I'm also going to make it clear that Kubernetes has this in alpha in 1.18.

File: content/en/news/releases/1.6.x/announcing-1.6/_index.md (hunk 22 above)
Comment (oaktowner) [reply content/en/news/releases/1.6.x/announcing-1.6/_index.md:29]: great suggestion. done

**Comment by frankbu** [comment]
Merged as part of PR #7275
